### PR TITLE
1699 - Clean Workplace 1 - Setup files

### PR DESCRIPTION
**Pull request by MarionJHolloway**
## Description
Trello ticket [#1699](https://trello.com/c/4OCfFf36/1699-clean-workplace-1-set-up-new-placeholder-esc-task-for-cleaning-workplace-data)

Set up Clean workspace job in polars

## Testing
- [x] Successful [Job / Step Function run](https://eu-west-2.console.aws.amazon.com/states/home?region=eu-west-2#/v2/executions/details/arn:aws:states:eu-west-2:856699698263:execution:1699-cw1-setup-Transform-ASCWDS-Data:6f4a51bd-19a9-4cd1-8b4b-03086c0ccaa1)

Only setting up placeholders, step function success is the only test

## Migration to polars checklist
- [x] Unit tests added
- [x] Docstrings added
- [x] Use new file structure when saving to S3
      `{dataset}_{sub_dataset_if_relevant}_{order_of_process_if_relevant}_{very_brief_description}`
- [x] Updated CHANGELOG
- [x] Moved Trello ticket to PR column

## Reviewer checklist
- [ ] PR solves the Trello ticket requirements
- [ ] Outputs appear reasonable and understandable
- [ ] The overall approach is appropriate and not over-engineered
- [ ] No obvious scalability, performance or interdependency concerns
- [ ] Tests appropriately cover the main behaviour/edge cases
- [ ] Naming and structure are broadly understandable and consistent
- [ ] Docstrings are sufficient for future maintenance
- [ ] Docstrings cover non-obvious behaviour

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -251,6 +251,8 @@ jobs:
             aws s3 sync "s3://sfc-main-datasets/domain=CQC/dataset=cqc_locations_04_full_cleaned_registered/" "s3://$BRANCH_DATASET_BUCKET/domain=CQC/dataset=cqc_locations_04_full_cleaned_registered/" --delete
             aws s3 sync "s3://sfc-main-datasets/domain=CQC/dataset=postcode_corrections/" "s3://$BRANCH_DATASET_BUCKET/domain=CQC/dataset=postcode_corrections/" --delete
             aws s3 sync "s3://sfc-main-datasets/domain=ONS/dataset=postcode_directory_cleaned/" "s3://$BRANCH_DATASET_BUCKET/domain=ONS/dataset=postcode_directory_cleaned/" --delete
+            aws s3 sync "s3://sfc-main-datasets/domain=ASCWDS/dataset=workplace/" "s3://$BRANCH_DATASET_BUCKET/domain=ASCWDS/dataset=workplace/" --delete
+            aws s3 sync "s3://sfc-main-datasets/domain=ASCWDS/dataset=worker/" "s3://$BRANCH_DATASET_BUCKET/domain=ASCWDS/dataset=worker/" --delete
```
Comment (RoyPriceCS50) [inline .circleci/config.yml:255]: Add a ticket to remove these from config after the job and validation are converted.

File: terraform/pipeline/step-functions/dynamic/Transform-ASCWDS-Data.json [hunk 2]
```diff
@@ -6,20 +6,69 @@
...
+                      "End": true
+                    }
+                  }
+                },
+                {
+                  "StartAt": "Clean ASCWDS Workplace polars",
+                  "States": {
+                    "Clean ASCWDS Workplace polars": {
+                      "Type": "Task",
+                      "Resource": "arn:aws:states:::ecs:runTask",
```
Comment (RoyPriceCS50) [inline terraform/pipeline/step-functions/dynamic/Transform-ASCWDS-Data.json]: runTask.sync

The GUI editor in AWS doesn't add that part it, and without it pipelines can fail without notification.

File: terraform/pipeline/step-functions/dynamic/Transform-ASCWDS-Data.json [hunk 3]
```diff
@@ -55,6 +104,46 @@
                     }
                   }
                 },
+                {
+                  "StartAt": "Validate Cleaned Workplace Polars",
+                  "States": {
+                    "Validate Cleaned Workplace Polars": {
+                      "Type": "Task",
+                      "Resource": "arn:aws:states:::ecs:runTask",
```
Comment (RoyPriceCS50) [inline terraform/pipeline/step-functions/dynamic/Transform-ASCWDS-Data.json]: runTask.sync

**Review by RoyPriceCS50 — APPROVED** [review]
Approved with tweak to the step function syntax, and remembering to remove raw data from sync after we're done with conversion.
